### DISCUSSION TITLE
Bagaimana cara agar logo di fontawesome dapat muncul

### DISCUSSION THREAD

**Discussion by jeremiaww** [teamalgoritma/community#85] [Q&A] [answered]
Hello,

Ingin bertanya bagaimana cara agar logo-logo dari fontawesome dapat muncul di shinyapps.
Sudah coba masukkan library nya, tapi tetap belum bisa muncul logo nya.
Seperti : "map-location", dan "people-group"

Terima kasih

**Comment by VictorNugraha** [comment, marked as answer]
Hallo, @jeremiaww !

Mengenai icon dari fontawesome yang ingin digunakan, sepertinya keyword untuk iconnya masih kurang tepat karena ketika saya coba dengan menggunakan fungsi **_icon()_** juga tidak muncul dan ketika saya coba memunculkan icon-nya dengan fungsi **_fa()_** dari **_library(fontawesome)_**, saya mendapatkan error message seperti di bawah ini.

<img width="501" alt="Capture2" src="https://user-images.githubusercontent.com/28379954/167264266-b23037c3-4507-4fda-9294-2bce141dfa31.PNG">

<img width="365" alt="Capture" src="https://user-images.githubusercontent.com/28379954/167264127-0ad4c8f8-99f1-4235-808d-accb537bc4cf.PNG">

Disini saya menyarankan Bapak untuk mencoba memilih beberapa keyword icon lainnya dari [link](https://fontawesome.com/v4/icons/) ini, semoga terdapat icon yang sesuai dengan kebutuhan Bapak.

_Additional info_ (mengutip dari dokumentasi fungsi **_icon()_**).  Ketika kita ingin menggunakan icon dari fontawesome, setiap prefix _**fa-**_ pada keyword icon jangan lupa untuk dihilangkan (Contoh: Nama keyword icon **"fa-calender"**, yang dituliskan hanyalah **"calender"**).

Demikian informasi yang dapat saya sampaikan, semoga menjawab pertanyaan yang diberikan dan selamat mencoba! :D

**Reply by jeremiaww** [reply]
Terima kasih mas
